docs/reference/intro.asciidoc

```diff
@@ -354,6 +369,11 @@ Does not yet support full-text search.
 | Enables users familiar with SQL to query {es} data using familiar syntax for BI and reporting.
 | <<sql-apis,`_sql`>>
 
+| {kibana-ref}/kuery-query.html[Kibana Query Language (KQL)]
+| Kibana Query Language (KQL) is a text-based query language for filtering data that is only available in the {kib} UI.
```

Comment:
`only` is probably inaccurate here because it's available through traditional es channels as well, right? 

bad edit: 

```suggestion
| Kibana Query Language (KQL) is a text-based query language for filtering data when you access it through the {kib} UI.
```